### PR TITLE
Time Varying Media Contribution Notebook

### PR DESCRIPTION
**Pull request by cetagostini**
## Related Issue

- [ ] Closes #
- [ ] Related to #

## Checklist

- [x] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [ ] Added necessary documentation (docstrings and/or example notebooks)
- [ ] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Modules affected

- [x] MMM
- [ ] CLV

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [x] Documentation
- [ ] Maintenance
- [ ] Other (please specify):

----
📚 Documentation preview 📚: https://pymc-marketing--778.org.readthedocs.build/en/778/

### CODE REVIEW

**Comment by cetagostini** [comment]
pre-commit.ci autofix

File: docs/source/notebooks/mmm/mmm_time_varying_media_example.ipynb
Comment (wd60622) [inline docs/source/notebooks/mmm/mmm_time_varying_media_example.ipynb:1]: Think you need to add this file to the index.md for the notebooks/mmm in docs

**Comment by cetagostini** [comment]
<p>Indeed but a little, real intercept is <code>6.</code> The recover <code>5.8</code>  <span class="ql-cursor">﻿</span></p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/778/discussion/#comment-2185274623'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by cetagostini** [comment]
<p>Good idea!</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/778/discussion/#comment-2185274620'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by juanitorduz** [comment]
<p>Can you experiment with the <code>drop_first</code> argument in the HSGP class? We might have redundant intercepts. </p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/778/discussion/#comment-2185274623'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by juanitorduz** [comment]
@cetagostini, very cool notebook! I left some comments :)

**Comment by wd60622** [comment]
<p>+1</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/778/discussion/#comment-2186337254'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by wd60622** [comment]
Let's add a new line (or combine lines) in the README to talk about this feature

File: docs/source/notebooks/mmm/mmm_example.ipynb [hunk 1]
```diff
@@ -9719,7 +9719,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.14"
+   "version": "3.11.8"
```
Comment (juanitorduz) [inline docs/source/notebooks/mmm/mmm_example.ipynb:9722]: Can we simply do not include the example mmm notebook in this PR 🙏 ?

**Comment by cetagostini** [comment]
pre-commit.ci autofix

**Comment by cetagostini** [comment]
<p>Done!</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/778/discussion/#comment-2186337254'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by cetagostini** [comment]
<p>Done!</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/778/discussion/#comment-2186779341'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by cetagostini** [comment]
<p>Done!</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/778/discussion/#comment-2186779400'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by cetagostini** [comment]
<p>done!</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/778/discussion/#comment-2186779467'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by cetagostini** [comment]
pre-commit.ci autofix

**Comment by cetagostini** [comment]
pre-commit.ci autofix

File: docs/source/notebooks/mmm/mmm_budget_allocation_example.ipynb [hunk 2]
```diff
@@ -1729,7 +1729,8 @@
    "name": "python3"
   },
   "language_info": {
-   "name": "python"
+   "name": "python",
+   "version": "3.10.14"
```
Comment (juanitorduz) [inline docs/source/notebooks/mmm/mmm_budget_allocation_example.ipynb:1733]: We can remove this notebook from the PR as there is no significant change

**Comment by cetagostini** [comment]
pre-commit.ci autofix

File: docs/source/notebooks/mmm/mmm_budget_allocation_example.ipynb (hunk 2 above)
Comment (cetagostini) [reply docs/source/notebooks/mmm/mmm_budget_allocation_example.ipynb]: Done!

**Comment by wd60622** [comment]
> Let's add a new line (or combine lines) in the README to talk about this feature

What do you think @cetagostini

File: docs/source/notebooks/mmm/mmm_time_varying_media_example.ipynb
Comment (wd60622) [inline docs/source/notebooks/mmm/mmm_time_varying_media_example.ipynb:1]: Only use a single H1 at the top. Or the documentation get messed up: 
https://pymc-marketing--778.org.readthedocs.build/en/778/notebooks/index.html

**Review by wd60622 — CHANGES_REQUESTED** [review]

**Comment by cetagostini** [comment]
pre-commit.ci autofix

**Review by wd60622 — APPROVED** [review]
Amazing

**Review by juanitorduz — APPROVED** [review]
Thanks @cetagostini !

**Pull request merged**